content/_data/changelogs/weekly.yml

```diff
@@ -4702,6 +4702,14 @@
 
     # pull: 4106 (PR title: Fix typo)
     # pull: 4089 (Internal: Prepare for using latest dom4j library.)
+- version: '2.186'
```

Comment:
We probably need to amend this once we have a fix for the SSH regression, but this works as a placeholder.